app/src/androidTest/java/org/mozilla/focus/activity/screenshots/ScreenGrabTest.java

```diff
@@ -260,12 +262,16 @@ private void takeAddToHomeScreenScreenshot() throws UiObjectNotFoundException {
         TestHelper.AddtoHSCancelBtn.waitForExists(waitingTime);
         Screengrab.screenshot("AddtoHSDialog");
         TestHelper.pressBackKey();
-        TestHelper.pressBackKey();
+
```

Comment:
Video shows that the simulator is slow enough that the keyboard isn't pulled when the dialog is opened. If Back key is pressed twice before the keyboard is visible, app will exit to the initial page.